tools/slang-capability-generator/capability-generator-main.cpp

```diff
@@ -18,18 +18,27 @@ namespace Diagnostics
 #undef DIAGNOSTIC
 }
 
+const Index kCapabilityDefCount = 3000;
+
 enum class CapabilityFlavor
 {
     Normal,
     Abstract,
     Alias
 };
+enum class AddCapabilityOptions
+{
+    Union = 0,
```

Comment:
Do we ever need `Union` in the generator? What happens if we only support inclusive join(|&) and ordinary join(+)?

In that case we can just repurpose | to mean inclusive join and delete this enum.

Reply:
We never needed union in the generator, Union is just a restricted 'inclusive join'.

I wonder if we should keep the enum just for 'ease of reading' (easier to change/add in the future if need-be)?

Reply:
We could, but my concern is that the `howToJoinWithNextSet` isn't a standard practice of defining expression trees. If we only have one way of joining, then the current syntax tree is clean and make sense, but having all terms appear in a single list that are joined in different ways depending on `list[i-1].howToJoinWithNextSet` is very weird and hard to understand.

Reply:
makes sense, `howToJoinWithNextSet` will be removed.